### PR TITLE
Stub out the mode change command

### PR DESCRIPTION
**Pull request by nomis52**
It doesn't do anything yet.

### CODE REVIEW

File: firmware/src/app.c [hunk 1]
```diff
@@ -31,6 +32,8 @@
 #include "transceiver.h"
 #include "usb_transport.h"
 
+const uint8_t OUR_UID[UID_LENGTH] = {0x7a, 0x70, 0xff, 0xff, 0xfe, 0};
```
Comment (peternewman) [inline firmware/src/app.c]: This should probably have a todo to move to a new range when Ja Rule gets a proper block allocated.

File: firmware/src/constants.h [hunk 2]
```diff
@@ -114,6 +114,12 @@ typedef enum {
    */
   COMMAND_RESET_DEVICE = 0x00,
 
+  /**
+   * @brief Change the operating mode of the device.
+   * See @ref message_cmd_set_mode.
```
Comment (peternewman) [inline firmware/src/constants.h]: Leave a blank line or @sa

**Comment by peternewman** [comment]
LGTM; fix Travis.

File: firmware/src/app.c (hunk 1 above)
Comment (nomis52) [inline firmware/src/app.c]: Added in the next commit

**Comment by nomis52** [comment]
Fixed

**Pull request merged**